Localize/LocProject.json

```diff
@@ -0,0 +1,31 @@
+{
+	"Projects": [
+	  {
+		"LanguageSet": "Azure_Languages",
+		"LocItems": [
+		  {
+			"SourceFile": "/Users/tjlambert/Documents/TurningOnOneLocBuild/xamarin-macios/tools/devops/../../msbuild/Xamarin.Localization.MSBuild/MSBStrings.resx",
```

Comment:
This file should have not been commited at all, should be generated by the makefile.